roles/ceph-facts/tasks/set_radosgw_address.yml

```diff
@@ -74,3 +74,4 @@
   when:
     - inventory_hostname in groups.get(rgw_group_name, [])
     - rgw_multisite | bool
+    - hostvars[item]["rgw_multisite"] | bool
```

Comment:
Why are you keeping both `rgw_multisite` and `hostvars[item]["rgw_multisite"]` ? the last one should be enough right ?